book/src/getting-started/index.md

```diff
@@ -9,19 +9,32 @@ SPDX-License-Identifier: MIT OR Apache-2.0
 
 In comparison to other Qt-Rust-bindings, CXX-Qt does not aim to simply expose Qt functionality to Rust, but rather to completely integrate Rust into the Qt ecosystem.
 
-In this guide we'll go through a minimal, but complete example that uses CXX-Qt to create your own QObject in Rust and integrate it with a small QML-based GUI.
-As CXX-Qt aims to integrate Rust into the existing Qt ecosystem, you should have basic knowledge of Qt and QML before attempting to follow this guide.
+In this guide we'll go through a minimal example that uses CXX-Qt to create your own QObject in Rust and integrate it with a small QML-based GUI.
+
+### Prerequisites
+This guide won't be able to explain everything to you, but it will try its best to make sure everyone can follow along.
+However, a few things you should make sure you're familiar with before attempting to follow this guide, as it may be confusing otherwise.
+
+First of all, you should be familiar with Rust. There are many great resources for learning Rust, like [the book](https://doc.rust-lang.org/book/).
+
+As CXX-Qt aims to integrate Rust into the existing Qt ecosystem, you should have basic knowledge of Qt and QML.
 If you're not familiar with Qt/QML yet, take a look at the [Qt Getting started guide](https://doc.qt.io/qt-5/gettingstarted.html) or the [QML intro](https://doc.qt.io/qt-5/qmlapplications.html) respectively.
```

Comment:
wonder if these should link to the Qt 6 pages now

Reply:
Good point.
If you're new to Qt, might as well go with Qt6.